### PR TITLE
zoom-us: 5.10.{4,6} -> 5.11.1

### PR DESCRIPTION
**Pull request by Maverickzero**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by necessarily-equal** [comment]
I wrote this patch that updates to 5.11.0.3540 and adds Linux screen sharing support on wayland. Should I make another PR for it or you want to cherry-pick?

```diff
diff --git a/pkgs/applications/networking/instant-messengers/zoom-us/default.nix b/pkgs/applications/networking/instant-messengers/zoom-us/default.nix
index 65c7942b5b3..1d73939b287 100644
--- a/pkgs/applications/networking/instant-messengers/zoom-us/default.nix
+++ b/pkgs/applications/networking/instant-messengers/zoom-us/default.nix
@@ -1,6 +1,7 @@
 { stdenv
 , lib
 , fetchurl
+, pipewire
 , makeWrapper
 , xar
 , cpio
@@ -46,7 +47,7 @@ let
   version = {
     aarch64-darwin = "5.11.0.8323";
     x86_64-linux = "5.11.0.8323";
-    x86_64-linux = "5.10.4.2845";
+    x86_64-linux = "5.11.0.3540";
    }.${system} or throwSystem;
 
   srcs = {
@@ -60,7 +61,7 @@ let
     };
     x86_64-linux = fetchurl {
       url = "https://zoom.us/client/${version}/zoom_x86_64.pkg.tar.xz";
-      sha256 = "9gspydrGaEjzAM0nK1u0XNm07HTupJ2wnPxCFWy+Nts=";
+      sha256 = "sha256-ruw6ZKZo4spY9H8yQmMYKJKRi0ZNXf1+BlhLqdPazIw=";
     };
   };
 
@@ -76,6 +77,7 @@ let
     expat
     libdrm
     libGL
+    pipewire
     fontconfig
     freetype
     gtk3
```

**Comment by Maverickzero** [comment]
> I wrote this patch that updates to 5.11.0.3540 and adds Linux screen sharing support on wayland. Should I make another PR for it or you want to cherry-pick?
> 
> ```diff
> diff --git a/pkgs/applications/networking/instant-messengers/zoom-us/default.nix b/pkgs/applications/networking/instant-messengers/zoom-us/default.nix
> index 65c7942b5b3..1d73939b287 100644
> --- a/pkgs/applications/networking/instant-messengers/zoom-us/default.nix
> +++ b/pkgs/applications/networking/instant-messengers/zoom-us/default.nix
> @@ -1,6 +1,7 @@
>  { stdenv
>  , lib
>  , fetchurl
> +, pipewire
>  , makeWrapper
>  , xar
>  , cpio
> @@ -46,7 +47,7 @@ let
>    version = {
>      aarch64-darwin = "5.11.0.8323";
>      x86_64-linux = "5.11.0.8323";
> -    x86_64-linux = "5.10.4.2845";
> +    x86_64-linux = "5.11.0.3540";
>     }.${system} or throwSystem;
>  
>    srcs = {
> @@ -60,7 +61,7 @@ let
>      };
>      x86_64-linux = fetchurl {
>        url = "https://zoom.us/client/${version}/zoom_x86_64.pkg.tar.xz";
> -      sha256 = "9gspydrGaEjzAM0nK1u0XNm07HTupJ2wnPxCFWy+Nts=";
> +      sha256 = "sha256-ruw6ZKZo4spY9H8yQmMYKJKRi0ZNXf1+BlhLqdPazIw=";
>      };
>    };
>  
> @@ -76,6 +77,7 @@ let
>      expat
>      libdrm
>      libGL
> +    pipewire
>      fontconfig
>      freetype
>      gtk3
> ```

Either is fine by me, maybe it is just simpler for you to make a PR? Then I can close this one. Does it already work for you btw? It still gives me an error about os support, since it reads the os name from /etc/os-release

**Comment by necessarily-equal** [comment]
It works for me if I explicitely edit `~/.config/zoomus.conf` to say `enableWaylandShare=true`

**Comment by necessarily-equal** [comment]
Can you do screen sharing on say, https://meet.webconf.ch/ ?

**Comment by Maverickzero** [comment]
> Can you do screen sharing on say, https://meet.webconf.ch/ ?

Screen sharing works fine if I use zoom on XWayland or in the browser. The issue with zoom wayland at this moment for me is that it still complains about the os. It is as if it does not honor the ~/.config/zoomus.conf

**Comment by necessarily-equal** [comment]
It may very well* make a difference. I would try setting 
XDG_CURRENT_DESKTOP="GNOME" and see if that's better.

*: https://github.com/flathub/us.zoom.Zoom/blob/master/zoom.sh#L21

**Comment by Maverickzero** [comment]
> It may very well* make a difference. I would try setting XDG_CURRENT_DESKTOP="GNOME" and see if that's better. *: https://github.com/flathub/us.zoom.Zoom/blob/master/zoom.sh#L21
> […](#)
> On Thu, Jun 23 2022 at 02:15:21 -07:00:00, Yuri A. Martinez Falcão ***@***.***> wrote: >> Can you do screen sharing on say, https://meet.webconf.ch/ ? >> > Screen sharing works fine if I use zoom on XWayland or in the > browser. The issue with zoom wayland at this moment for me is that > it still complains about the os. It is as if it does not honor the > ~/.config/zoomus.conf > It might be good to say that I'm on sway, not on Gnome, do you think that it might make a difference? — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you commented.Message ID: ***@***.***>

That was it! You sir just fixed my zoom screen sharing woes!

**Comment by Maverickzero** [comment]
> I'd like this PR to be merged with #178597 .

I just merged from that zoom darwin PR

**Comment by doronbehar** [comment]
Does anyone else experience a crash when trying to share their screen? Sharing my log here:

```
ZoomLauncher started.
Zoom path is: /nix/store/5aigasm0j0kzykwa35yqahinf3l46214-zoom-5.11.0.3540/opt/zoom
cmd line: 
Start subprocess: /nix/store/5aigasm0j0kzykwa35yqahinf3l46214-zoom-5.11.0.3540/opt/zoom/zoom sucessfully,  process pid: 23987 
No PulseAudio daemon running, or not running as session daemon.
                             Class      App      Lib Possible Culprit Flags
                resip::Connection      648      648 
                      resip::Data       36       36 
                 resip::DnsResult     1080     1080 
                   resip::Headers        1        1 
          resip::MsgHeaderScanner       40       40 
                resip::SipMessage     5208     5208 
         resip::TransportSelector      864      864 
                     resip::Tuple      128      128 
              resip::UdpTransport     1144     1144 
          resip::GenericIPAddress       28       28 

zoom started.
No PulseAudio daemon running, or not running as session daemon.
No PulseAudio daemon running, or not running as session daemon.
                                                        Class    Class        App    App        Lib   Possible Culprit FlagsLib
  Possible Culprit Flags 
                            resip:: Connection  resip::Connection     648       648     648       648
  
                                         resip:: Data  resip::      36Data        36      36  
      36   
                              resip:: DnsResult  resip::    1080DnsResult      1080    1080  
    1080   
                                   resip:: Headersresip:: Headers       1         1       1         1
  
                  resip:: MsgHeaderScannerresip:: MsgHeaderScanner      40        40      40        40
  
                              resip:: SipMessageresip:: SipMessage    5208      5208    5208      5208
  
                resip:: TransportSelectorresip:: TransportSelector     864       864     864       864
  
                                        resip:: Tupleresip:: Tuple     128       128     128       128
  
                          resip:: UdpTransportresip:: UdpTransport    1144      1144    1144      1144
  
                 resip:: GenericIPAddress  resip::      28GenericIPAddress        28      28  
      28
 

zoom started.
zoom started.
Client: Breakpad is using Single Client Mode! client fd = -1
Warning: Ignoring XDG_SESSION_TYPE=wayland on Gnome. Use QT_QPA_PLATFORM=wayland to run on Wayland anyway.
QApplication: invalid style override passed, ignoring it.
    Available styles: Windows, Fusion
No PulseAudio daemon running, or not running as session daemon.
[CZPClientLogMgr::LogClientEnvironment] [MacAddr: DC:FB:48:7A:62:69][client: Linux][OS:  NixOS 22.11 Raccoon x64][Hardware: CPU Core:4 Frenquency:1.8 G Memory size:15823MB CPU Brand:Intel(R) Core(TM) i7-8565U CPU @ 1.80GHz GPU Brand:][Req ID: ]
                             Class      App      Lib Possible Culprit Flags
                resip::Connection      648      648 
                      resip::Data       36       36 
                 resip::DnsResult     1080     1080 
                   resip::Headers        1        1 
          resip::MsgHeaderScanner       40       40 
                resip::SipMessage     5208     5208 
         resip::TransportSelector      864      864 
                     resip::Tuple      128      128 
              resip::UdpTransport     1144     1144 
          resip::GenericIPAddress       28       28 

Linux Client Version is 5.11.0 (3540)
QSG_RENDER_LOOP is 
XDG_CURRENT_DESKTOP = GNOME;   GDMSESSION = gnome;   XDG_SESSION_TYPE = wayland
zoom started.
Graphics Card Info:: 00:02.0 VGA compatible controller: Intel Corporation WhiskeyLake-U GT2 [UHD Graphics 620] (rev 02)
Zoom package arch is 64bit, runing OS arch is x86_64
Using Wayland-EGL
qt.scenegraph.general: threaded render loop
qt.scenegraph.general: Using sg animation driver
qt.scenegraph.general: Animation Driver: using vsync: 16.67 ms
Using the 'xdg-shell' shell integration
xkbcommon: ERROR: couldn't find a Compose file for locale "en_IL.UTF-8" (mapped to "en_IL.UTF-8")
qt.qpa.wayland: Wayland does not support QWindow::requestActivate()
qt.scenegraph.general: Using sg animation driver
qt.scenegraph.general: Animation Driver: using vsync: 16.67 ms
qt.scenegraph.general: texture atlas dimensions: 1024x512
qt.scenegraph.general: R/G/B/A Buffers:   8 8 8 8
qt.scenegraph.general: Depth Buffer:      24
qt.scenegraph.general: Stencil Buffer:    8
qt.scenegraph.general: Samples:           0
qt.scenegraph.general: GL_VENDOR:         Intel
qt.scenegraph.general: GL_RENDERER:       Mesa Intel(R) UHD Graphics 620 (WHL GT2)
qt.scenegraph.general: GL_VERSION:        4.6 (Compatibility Profile) Mesa 22.0.4
qt.scenegraph.general: GL_EXTENSIONS:    GL_3DFX_texture_compression_FXT1 GL_AMD_conservative_depth GL_AMD_depth_clamp_separate GL_AMD_draw_buffers_blend GL_AMD_multi_draw_indirect GL_AMD_performance_monitor GL_AMD_pinned_memory GL_AMD_query_buffer_object GL_AMD_seamless_cubemap_per_texture GL_AMD_shader_stencil_export GL_AMD_shader_trinary_minmax GL_AMD_texture_texture4 GL_AMD_vertex_shader_layer GL_AMD_vertex_shader_viewport_index GL_ANGLE_texture_compression_dxt3 GL_ANGLE_texture_compression_dxt5 GL_APPLE_packed_pixels GL_ARB_ES2_compatibility GL_ARB_ES3_1_compatibility GL_ARB_ES3_2_compatibility GL_ARB_ES3_compatibility GL_ARB_arrays_of_arrays GL_ARB_base_instance GL_ARB_blend_func_extended GL_ARB_buffer_storage GL_ARB_clear_buffer_object GL_ARB_clear_texture GL_ARB_clip_control GL_ARB_color_buffer_float GL_ARB_compatibility GL_ARB_compressed_texture_pixel_storage GL_ARB_compute_shader GL_ARB_compute_variable_group_size GL_ARB_conditional_render_inverted GL_ARB_conservative_depth GL_ARB_copy_buffer GL_ARB_copy_image GL_ARB_cull_distance GL_ARB_debug_output GL_ARB_depth_buffer_float GL_ARB_depth_clamp GL_ARB_depth_texture GL_ARB_derivative_control GL_ARB_direct_state_access GL_ARB_draw_buffers GL_ARB_draw_buffers_blend GL_ARB_draw_elements_base_vertex GL_ARB_draw_indirect GL_ARB_draw_instanced GL_ARB_enhanced_layouts GL_ARB_explicit_attrib_location GL_ARB_explicit_uniform_location GL_ARB_fragment_coord_conventions GL_ARB_fragment_layer_viewport GL_ARB_fragment_program GL_ARB_fragment_program_shadow GL_ARB_fragment_shader GL_ARB_fragment_shader_interlock GL_ARB_framebuffer_no_attachments GL_ARB_framebuffer_object GL_ARB_framebuffer_sRGB GL_ARB_get_program_binary GL_ARB_get_texture_sub_image GL_ARB_gl_spirv GL_ARB_gpu_shader5 GL_ARB_gpu_shader_fp64 GL_ARB_gpu_shader_int64 GL_ARB_half_float_pixel GL_ARB_half_float_vertex GL_ARB_indirect_parameters GL_ARB_instanced_arrays GL_ARB_internalformat_query GL_ARB_internalformat_query2 GL_ARB_invalidate_subdata GL_ARB_map_buffer_alignment GL_ARB_map_buffer_range GL_ARB_multi_bind GL_ARB_multi_draw_indirect GL_ARB_multisample GL_ARB_multitexture GL_ARB_occlusion_query GL_ARB_occlusion_query2 GL_ARB_parallel_shader_compile GL_ARB_pipeline_statistics_query GL_ARB_pixel_buffer_object GL_ARB_point_parameters GL_ARB_point_sprite GL_ARB_polygon_offset_clamp GL_ARB_post_depth_coverage GL_ARB_program_interface_query GL_ARB_provoking_vertex GL_ARB_query_buffer_object GL_ARB_robust_buffer_access_behavior GL_ARB_robustness GL_ARB_sample_shading GL_ARB_sampler_objects GL_ARB_seamless_cube_map GL_ARB_seamless_cubemap_per_texture GL_ARB_separate_shader_objects GL_ARB_shader_atomic_counter_ops GL_ARB_shader_atomic_counters GL_ARB_shader_ballot GL_ARB_shader_bit_encoding GL_ARB_shader_clock GL_ARB_shader_draw_parameters GL_ARB_shader_group_vote GL_ARB_shader_image_load_store GL_ARB_shader_image_size GL_ARB_shader_objects GL_ARB_shader_precision GL_ARB_shader_stencil_export GL_ARB_shader_storage_buffer_object GL_ARB_shader_subroutine GL_ARB_shader_texture_image_samples GL_ARB_shader_texture_lod GL_ARB_shader_viewport_layer_array GL_ARB_shading_language_100 GL_ARB_shading_language_420pack GL_ARB_shading_language_include GL_ARB_shading_language_packing GL_ARB_shadow GL_ARB_spirv_extensions GL_ARB_stencil_texturing GL_ARB_sync GL_ARB_tessellation_shader GL_ARB_texture_barrier GL_ARB_texture_border_clamp GL_ARB_texture_buffer_object GL_ARB_texture_buffer_object_rgb32 GL_ARB_texture_buffer_range GL_ARB_texture_compression GL_ARB_texture_compression_bptc GL_ARB_texture_compression_rgtc GL_ARB_texture_cube_map GL_ARB_texture_cube_map_array GL_ARB_texture_env_add GL_ARB_texture_env_combine GL_ARB_texture_env_crossbar GL_ARB_texture_env_dot3 GL_ARB_texture_filter_anisotropic GL_ARB_texture_float GL_ARB_texture_gather GL_ARB_texture_mirror_clamp_to_edge GL_ARB_texture_mirrored_repeat GL_ARB_texture_multisample GL_ARB_texture_non_power_of_two GL_ARB_texture_query_levels GL_ARB_texture_query_lod GL_ARB_texture_rectangle GL_ARB_texture_rg GL_ARB_texture_rgb10_a2ui GL_ARB_texture_stencil8 GL_ARB_texture_storage GL_ARB_texture_storage_multisample GL_ARB_texture_swizzle GL_ARB_texture_view GL_ARB_timer_query GL_ARB_transform_feedback2 GL_ARB_transform_feedback3 GL_ARB_transform_feedback_instanced GL_ARB_transform_feedback_overflow_query GL_ARB_transpose_matrix GL_ARB_uniform_buffer_object GL_ARB_vertex_array_bgra GL_ARB_vertex_array_object GL_ARB_vertex_attrib_64bit GL_ARB_vertex_attrib_binding GL_ARB_vertex_buffer_object GL_ARB_vertex_program GL_ARB_vertex_shader GL_ARB_vertex_type_10f_11f_11f_rev GL_ARB_vertex_type_2_10_10_10_rev GL_ARB_viewport_array GL_ARB_window_pos GL_ATI_blend_equation_separate GL_ATI_draw_buffers GL_ATI_fragment_shader GL_ATI_separate_stencil GL_ATI_texture_env_combine3 GL_ATI_texture_float GL_EXT_EGL_image_storage GL_EXT_EGL_sync GL_EXT_abgr GL_EXT_bgra GL_EXT_blend_color GL_EXT_blend_equation_separate GL_EXT_blend_func_separate GL_EXT_blend_minmax GL_EXT_blend_subtract GL_EXT_compiled_vertex_array GL_EXT_copy_texture GL_EXT_demote_to_helper_invocation GL_EXT_direct_state_access GL_EXT_draw_buffers2 GL_EXT_draw_instanced GL_EXT_draw_range_elements GL_EXT_fog_coord GL_EXT_framebuffer_blit GL_EXT_framebuffer_multisample GL_EXT_framebuffer_multisample_blit_scaled GL_EXT_framebuffer_object GL_EXT_framebuffer_sRGB GL_EXT_gpu_program_parameters GL_EXT_gpu_shader4 GL_EXT_memory_object GL_EXT_memory_object_fd GL_EXT_multi_draw_arrays GL_EXT_packed_depth_stencil GL_EXT_packed_float GL_EXT_packed_pixels GL_EXT_pixel_buffer_object GL_EXT_point_parameters GL_EXT_polygon_offset_clamp GL_EXT_provoking_vertex GL_EXT_rescale_normal GL_EXT_secondary_color GL_EXT_semaphore GL_EXT_semaphore_fd GL_EXT_separate_specular_color GL_EXT_shader_framebuffer_fetch GL_EXT_shader_framebuffer_fetch_non_coherent GL_EXT_shader_integer_mix GL_EXT_shader_samples_identical GL_EXT_shadow_funcs GL_EXT_stencil_two_side GL_EXT_stencil_wrap GL_EXT_subtexture GL_EXT_texture GL_EXT_texture3D GL_EXT_texture_array GL_EXT_texture_buffer_object GL_EXT_texture_compression_dxt1 GL_EXT_texture_compression_rgtc GL_EXT_texture_compression_s3tc GL_EXT_texture_cube_map GL_EXT_texture_edge_clamp GL_EXT_texture_env_add GL_EXT_texture_env_combine GL_EXT_texture_env_dot3 GL_EXT_texture_filter_anisotropic GL_EXT_texture_integer GL_EXT_texture_lod_bias GL_EXT_texture_object GL_EXT_texture_rectangle GL_EXT_texture_sRGB GL_EXT_texture_sRGB_R8 GL_EXT_texture_sRGB_decode GL_EXT_texture_shadow_lod GL_EXT_texture_shared_exponent GL_EXT_texture_snorm GL_EXT_texture_swizzle GL_EXT_timer_query GL_EXT_transform_feedback GL_EXT_vertex_array GL_EXT_vertex_array_bgra GL_EXT_vertex_attrib_64bit GL_IBM_multimode_draw_arrays GL_IBM_rasterpos_clip GL_IBM_texture_mirrored_repeat GL_INGR_blend_func_separate GL_INTEL_blackhole_render GL_INTEL_performance_query GL_INTEL_shader_atomic_float_minmax GL_INTEL_shader_integer_functions2 GL_KHR_blend_equation_advanced GL_KHR_blend_equation_advanced_coherent GL_KHR_context_flush_control GL_KHR_debug GL_KHR_no_error GL_KHR_parallel_shader_compile GL_KHR_robust_buffer_access_behavior GL_KHR_robustness GL_KHR_texture_compression_astc_ldr GL_KHR_texture_compression_astc_sliced_3d GL_MESA_framebuffer_flip_y GL_MESA_pack_invert GL_MESA_shader_integer_functions GL_MESA_texture_signed_rgba GL_MESA_window_pos GL_NV_blend_square GL_NV_compute_shader_derivatives GL_NV_conditional_render GL_NV_copy_depth_to_color GL_NV_copy_image GL_NV_depth_clamp GL_NV_fog_distance GL_NV_fragment_shader_interlock GL_NV_half_float GL_NV_light_max_exponent GL_NV_packed_depth_stencil GL_NV_primitive_restart GL_NV_texgen_reflection GL_NV_texture_barrier GL_NV_texture_env_combine4 GL_NV_texture_rectangle GL_OES_EGL_image GL_OES_read_format GL_S3_s3tc GL_SGIS_generate_mipmap GL_SGIS_texture_border_clamp GL_SGIS_texture_edge_clamp GL_SGIS_texture_lod GL_SUN_multi_draw_arrays
qt.scenegraph.general: Max Texture Size: 16384
qt.scenegraph.general: Debug context:    false
AppIconMgr::systemDesktopName log Desktop Name: gnome 
Error: Send error, 22 Invalid argument
Error: Send error, 22 Invalid argument
Error: Send error, 22 Invalid argument
Unsupported modifier, resource creation failed.
XXX: resource creation failed
qt.scenegraph.general: Using sg animation driver
qt.scenegraph.general: Animation Driver: using vsync: 16.67 ms
qt.scenegraph.general: texture atlas dimensions: 2048x1024
qt.qpa.wayland: Wayland does not support QWindow::requestActivate()
qt.scenegraph.general: Using sg animation driver
qt.scenegraph.general: Animation Driver: using vsync: 16.67 ms
qt.scenegraph.general: texture atlas dimensions: 2048x1024
qt.scenegraph.general: Using sg animation driver
qt.scenegraph.general: Animation Driver: using vsync: 16.67 ms
qt.scenegraph.general: texture atlas dimensions: 512x512
Unsupported modifier, resource creation failed.
XXX: resource creation failed
qt.scenegraph.general: Using sg animation driver
qt.scenegraph.general: Animation Driver: using vsync: 16.67 ms
qt.scenegraph.general: texture atlas dimensions: 512x512
qt.qpa.wayland: Wayland does not support QWindow::requestActivate()
qt.scenegraph.general: Using sg animation driver
qt.scenegraph.general: Animation Driver: using vsync: 16.67 ms
Unsupported modifier, resource creation failed.
XXX: resource creation failed
qt.scenegraph.general: texture atlas dimensions: 1024x1024
Unsupported modifier, resource creation failed.
XXX: resource creation failed
Unsupported modifier, resource creation failed.
XXX: resource creation failed
This plugin does not support setting window opacity
This plugin does not support setting window opacity
This plugin does not support setting window opacity
This plugin does not support setting window opacity
This plugin does not support setting window opacity
This plugin does not support setting window opacity
This plugin does not support setting window opacity
This plugin does not support setting window opacity
This plugin does not support setting window opacity
This plugin does not support setting window opacity
This plugin does not support setting window opacity
This plugin does not support setting window opacity
This plugin does not support setting window opacity
This plugin does not support setting window opacity
This plugin does not support setting window opacity
This plugin does not support setting window opacity
This plugin does not support setting window opacity
This plugin does not support setting window opacity
This plugin does not support setting window opacity
This plugin does not support setting window opacity
This plugin does not support setting window opacity
This plugin does not support setting window opacity
This plugin does not support setting window opacity
This plugin does not support setting window opacity
This plugin does not support setting window opacity
This plugin does not support setting window opacity
This plugin does not support setting window opacity
This plugin does not support setting window opacity
This plugin does not support setting window opacity
This plugin does not support setting window opacity
This plugin does not support setting window opacity
This plugin does not support setting window opacity
This plugin does not support setting window opacity
This plugin does not support setting window opacity
This plugin does not support setting window opacity
This plugin does not support setting window opacity
This plugin does not support setting window opacity
This plugin does not support setting window opacity
This plugin does not support setting window opacity
This plugin does not support setting window opacity
No PulseAudio daemon running, or not running as session daemon.
zoom was exited due to a handled signal: 7 
ZoomLauncher exit.
```

**Comment by necessarily-equal** [comment]
re: crash, it sounds like you may be missing pulseaudio support. In 
case it's not that I fear NixOS couldn't do much about it since it's 
proprietary software.

**Comment by doronbehar** [comment]
> re: crash, it sounds like you may be missing pulseaudio support.

I don't set `pulseaudioSupport = false;` when I call the derivation if that's what you mean. Do you take any action to enable pulseaudio support?

**Comment by doronbehar** [comment]
For the record, I don't think screen sharing has ever worked for me, so I don't vetoe this PR. I still wish to get screen sharing not crash it though.

**Comment by Maverickzero** [comment]
> @Maverickzero the Darwin release had a bug fix, could you merge again from whiteley:zoom.us-darwin-5.11.0.8323 please? Thanks!

Hi Whiteley, I just pulled again from your branch with the updates, thanks for the headsup!

File: pkgs/applications/networking/instant-messengers/zoom-us/default.nix [hunk 1]
```diff
@@ -44,23 +45,23 @@ let
 
   # Zoom versions are released at different times for each platform
   version = {
-    aarch64-darwin = "5.10.4.6592";
-    x86_64-darwin = "5.10.4.6592";
-    x86_64-linux = "5.10.6.3192";
+    aarch64-darwin = "5.11.1.8356";
+    x86_64-darwin = "5.11.1.8356";
+    x86_64-linux = "5.11.0.3540";
```
Comment (doronbehar) [inline pkgs/applications/networking/instant-messengers/zoom-us/default.nix:50]: According to https://zoom.us/download?os=linux there's a 5.11.1 Linux version.

**Pull request merged**